docs/walletkit/react-native/chain-abstraction.mdx

```diff
@@ -0,0 +1,180 @@
+# Chain Abstraction
+
+Chain Abstraction in WalletKit enables users with stablecoins on any network to spend them on-the-fly on a different network. Our Chain Abstraction solution provides a toolkit for wallet developers to integrate this complex functionality using WalletKit.
+
+For example, when an app requests a 100 USDC payment on Base network but the user only has USDC on Arbitrum, WalletKit offers methods to detect this mismatch, generate necessary transactions, track the cross-chain transfer, and complete the original transaction after bridging finishes.
+
+:::note
+Chain abstraction is currently in experimental phase and requires the `@ChainAbstractionExperimentalApi` annotation.
```

Comment:
@ganchoradkov `@ChainAbstractionExperimentalApi` what annotation do you use in RN?

Reply:
with JSDoc annotation
```
/**
 * @experimental
 * This method is experimental and may change in the future.
 */
```
this shows the warning when used in IDE